idp-service/src/test/java/io/mosip/idp/services/AuthorizationServiceTest.java

```diff
@@ -248,7 +217,7 @@ public void getOauthDetails_withValidClaimsInDbAndInValidClaimsInReq() throws Ex
     }
 
     @Test
-    public void getOauthDetails_withNullAcrInDB() throws Exception {
+    public void getOauthDetails_withNullAcrInDB_thenFail() throws Exception {
```

Comment:
thenPass and thenFail is new convention ?

Reply:
I had followed this already in merged test cases, so followed the same here.